modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -254,6 +255,19 @@ def evss_service
       ClaimsApi::EVSSService::Base.new
     end
 
+    def fes_service
+      ClaimsApi::FesService::Base.new
+    end
+
+    # Returns the appropriate submission service based on feature flags
+    def submission_service
+      if Flipper.enabled? :claims_api_v1_lh_fes_auto_establish_claim_enabled
```

Comment:
This should be :claims_api_v2_lh_fes_auto_establish_claim_enabled for the v2 flow.